python/nav/portadmin/snmputils.py

```diff
@@ -658,6 +680,19 @@ def disable_cisco_voice_vlan(self, interface):
             _logger.error('Error disabling voice vlan: %s', error)
 
         return status
+        
+    def disable_cisco_cdp(self, interface):
+        """Set CDP using Cisco specific oid"""
+        status = None
+        try:
+            status = self._set_netbox_value(
+                self.cdp_oid, interface.ifindex, 'i', 2)
+        except SnmpError as error:
+            _logger.error('Error setting cdp on interface: %s', error)
+        except ValueError as error:
+            _logger.error('%s is not a valid option for cdp', Integer32(2))
```

Comment:
There is no `Integer32` symbol here, this would fail. How about just using `2`?

Reply:
Yes, I guess you're right. I copied that from somewhere and didn't check. I just pushed a new commit.